docs/content/installation/installation-with-operator.md

```diff
@@ -37,7 +37,7 @@ spec:
     image:
       pullPolicy: IfNotPresent
       repository: nginx/nginx-ingress
-      tag: 3.0.1-ubi
+      tag: 3.0.2-ubi
```

Comment:
Not directly related to these changes, but the operator tags above on lines 20 and 21 are wrong - they should be v1.3.0 not v1.2.1